src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -102,6 +102,9 @@
 
     <!-- Defines the default document type property used when adding properties in the back-office (if missing or empty, defaults to Textstring -->
     <defaultDocumentTypeProperty>Textstring</defaultDocumentTypeProperty>
+    
+    <!-- Enable inherited document types. This feature will be deprecated in v8 -->

```

Comment:
this will essentially be a hidden config item so let's keep it out of all config files (i'll remove)